config.js

```diff
@@ -17,10 +19,21 @@ exports.client = {
 }
 
 exports.github = {
+  'dev': {
+    'teamId': '1940407', // github.com/dvcoders-test
+    'apiKey': process.env.GITHUB_API_KEY_DEV || null
```

Comment:
Do we really need to set this new env variable?
If my account is authorized in the test team, I can use my personal token to access API, so I do not think we need to make another env variable.